docker-entrypoint.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash -ex
+
+if [[ ! -d "$DATA_DIR" ]]; then
+        exit 1
+fi
+
+chown $UID:$GID /opt/hoover
```

Comment:
Isn't this already set in the Dockerfile? Is this used as a direct mount point anywhere?